config.js

```diff
@@ -422,6 +422,12 @@ var config = {
     // value will be used when the quality level is selected using "Manage Video Quality" slider.
     // startLastN: 1,
 
+    // Provides a way to control the bandwidth through the UI. Possible values:
+    // - negative ones that are ignored and estimations are used,
+    // - positive ones that will try to force the client by disabling estimations,
+    // - 0 that means no videos.
+    // assumedBandwidthBps: -1,
```

Comment:
Is this setting read from LJM? Do we want it like so or just the API value?

Reply:
Yes https://github.com/jitsi/lib-jitsi-meet/blob/731e6fae2ff11b06db2ec24db3aeeb017f6585be/modules/qualitycontrol/ReceiveVideoController.js#L184

Reply:
I figured we could also provide it in the config, but if it doesn't make sense, I can remove it.

Reply:
I think it's best if we limit it to API usage only.